.continuous-integration/travis/setup_dependencies_common.sh

```diff
@@ -53,9 +53,9 @@ fi
 # build_sphinx needs sphinx as well as matplotlib and wcsaxes (for plot_directive). 
 if [[ $SETUP_CMD == build_sphinx* ]]
 then
-  $CONDA_INSTALL Sphinx Pygments matplotlib
-  pip install wcsaxes
+  $CONDA_INSTALL matplotlib
   pip install sphinx_rtd_theme
```

Comment:
what about trying the same as before but with sphinx_rtd_theme in the conda command? (it's available in conda)